app/screens/settings-screen/account/account-screen.tsx

```diff
@@ -0,0 +1,61 @@
+import { makeStyles } from "@rneui/themed"
+import { ScrollView } from "react-native-gesture-handler"
+
+import { useI18nContext } from "@app/i18n/i18n-react"
+import { useLevel } from "@app/graphql/level-context"
+
+import { AccountDeleteContextProvider } from "./account-delete-context"
+import { Screen } from "@app/components/screen"
+import { AccountHeader } from "./header"
+import { AccountId } from "./id"
+import { SettingsGroup } from "../group"
+
+import { EmailSetting } from "./settings/email"
+import { PhoneSetting } from "./settings/phone"
+import { DangerZoneSettings } from "./settings/danger-zone"
+import { UpgradeTrialAccount } from "./settings/upgrade-trial-account"
+import { UpgradeAccountLevelOne } from "./settings/upgrade"
+import { TotpSetting } from "../totp"
+
+export const AccountScreen: React.FC = () => {
+  const styles = useStyles()
+  const { LL } = useI18nContext()
+
+  const { isAtLeastLevelOne } = useLevel()
+
+  const items = {
```

Comment:
`items` feel very/too generic for a variable name

Reply:
if only `authenticationMethods` and `misc` is been passed down throught the props, we probably don't need to defined a high level items.